ide/git/src/org/netbeans/modules/git/ui/shelve/ShelveChangesAction.java

```diff
@@ -294,18 +300,20 @@ public void setDisplayName (String displayName) {
         }
     }
     
+    // TODO git unstash in shelve action?
+    
     @NbBundle.Messages({
         "CTL_UnstashMenu.name=&Git Unstash",
         "CTL_UnstashMenu.name.popup=Git Unstash",
         "# {0} - stash index", "# {1} - stash name", "CTL_UnstashAction.name={0} - {1}"
     })
-    private static class UnshelveMenu {
+    private static class UnstashMenu {
```

Comment:
this might indicate that things were copied around. Git unstash should not be in the shelve action.

this lead also to this:
https://github.com/apache/netbeans/blob/e2dce5682f9aad540cafdf169d485061b4f3eaaf/ide/versioning.util/src/org/netbeans/modules/versioning/shelve/impl/ShelveChangesMenu.java#L152-L153
which is not the patch storage, but the git unstash action (unshelve is actually registered a few lines before that)

but this would be for other PRs.